biz.aQute.bndlib/src/aQute/bnd/build/DownloadBlocker.java

```diff
@@ -110,4 +110,8 @@ public File getFile() {
 	public String toString() {
 		return "DownloadBlocker [stage=" + stage + ", failure=" + failure + ", file=" + file + "]";
 	}
+
+	public String getPath() {
+		return file.getAbsolutePath();
+	}
```

Comment:
This cannot work. The whole point is that the blocker does not have the file until sometime in the future. You cannot access the file field until getReason has been called.